extensions/hibernate-orm/deployment/src/main/resources/dev-ui/hibernate-orm/hibernate-orm-persistence-units.js

```diff
@@ -0,0 +1,119 @@
+import { notifier } from 'notifier';
+
+export class HibernateOrmPersistenceUnitsComponent extends LitElement {
+
+    static styles = css`
+        .full-height {
+          height: 100%;
+        }
+        .ddl-script {
+            padding: 5px;
+        }
+        a.script-heading {
+            display: block;
+            float:left;
+            width: 90%;
+            text-decoration: none;
+        }
+    `;
+
+    jsonRpc = new JsonRpc("Hibernate ORM");
+
+    static properties = {
+        "_persistenceUnits": {state: true, type: Array}
+    }
+
+    connectedCallback() {
+        super.connectedCallback();
+        this.jsonRpc.getInfo().then(response => {
+            this._persistenceUnits = [];
+            response.result.persistenceUnits.forEach(pu => {
+                this._persistenceUnits.push(pu);
+            });
+        });
+    }
+
+    render() {
+        return html`${until(this._renderAllPUs(), html`<span>Loading...</span>`)}`;
+    }
+
+    _renderAllPUs() {
+        if (this._persistenceUnits) {
+            return this._persistenceUnits.length == 0
+                ? html`<p>No persistence units were found.</p>`
+                : html`
+                    <vaadin-tabsheet class="full-height">
+                        <span slot="prefix">Persistence Unit</span>
+                        <vaadin-tabs slot="tabs">
+                            ${this._persistenceUnits.map((pu) => html`
+                                <vaadin-tab id="pu-${pu.name}-persistence-unit">
+                                    <span>${pu.name}</span>
+                                </vaadin-tab>
+                                `)}
+                        </vaadin-tabs>
+
+                        ${this._persistenceUnits.map((pu) => html`
+                            <div class="full-height" tab="pu-${pu.name}-persistence-unit">
+                                ${this._renderPersistenceUnit(pu)}
+                            </div>
+                            `)}
+                    </vaadin-tabsheet>`;
+        }
+    }
+
+    _renderPersistenceUnit(pu) {
+        return html`
+                <vaadin-details>
+                    <vaadin-details-summary slot="summary" theme="filled">
+                        <vaadin-horizontal-layout
+                                theme="spacing"
+                                style="align-items: center;">
+                            <span>Create Script</span>
+                            <vaadin-button @click="${(e) => this._copyToClipboard(e, 'Create Script')}"
+                                    theme="small">
+                                <vaadin-icon icon="font-awesome-solid:clipboard"></vaadin-icon>
+                                Copy
+                            </vaadin-button>
+                        </vaadin-horizontal-layout>
+                    </vaadin-details-summary>
+                    <pre class="ddl-script">${pu.createDDL}</pre>
```

Comment:
We can probably use codemirror here to get code syntax highlighting. But I'll can add that as a follow up PR as the codemirror sql lib needs to be pulled in